n3fit/src/n3fit/scripts/n3fit_exec.py

```diff
@@ -124,7 +119,9 @@ def from_yaml(cls, o, *args, **kwargs):
         except yaml.error.YAMLError as e:
             raise ConfigError(f"Failed to parse yaml file: {e}")
         if not isinstance(file_content, dict):
-            raise ConfigError(f"Expecting input runcard to be a mapping, " f"not '{type(file_content)}'.")
+            raise ConfigError(
+                f"Expecting input runcard to be a mapping, " f"not '{type(file_content)}'."
```

Comment:
```suggestion
                f"Expecting input runcard to be a mapping, not '{type(file_content)}'."
```

Could you check if the changes done by black make sense? If there is some part that should not have been changed you can use `#fmt: on/off`